AppUpdater.swift

```diff
@@ -17,6 +17,8 @@ public class AppUpdater {
     var slug: String {
         return "\(owner)/\(repo)"
     }
+    
+    public var prerelease = false
```

Comment:
Not sure about the best language to use, e.g. should it be `updater.prerelease` or more like `updater.includePrereleases` or `updater.allowPrerelease`?

Reply:
I think `allowPrereleases` is most Swifty.